### PR TITLE
Adityaagr99 patch 1

### PR DESCRIPTION
**Pull request by adityaagr99**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -133,7 +133,8 @@ <h1 class="wow fadeInUp" data-wow-delay="1.6s" style="text-shadow:4px 4px 0 #242
                     <h3 class="wow bounceIn" data-wow-delay="0.9s" style="text-shadow:4px 4px 0 #242424">Feb 17 2018, ISB Hyderabad</h3>
                     <a href="#overview" class="btn btn-lg btn-default smoothScroll wow fadeInUp hidden-xs" data-wow-delay="1.6s">LEARN MORE</a>
                     <a href="https://www.townscript.com/e/startup-master-class-zero-to-one-take-a-leap-441222" class="btn btn-lg btn-danger smoothScroll wow fadeInUp" data-wow-delay="1.6s">Book Tickets</a>
-                    <h5 class="" style="margin-top:30px; text-shadow:2px 2px 0 #242424">An initiative of Alumni Association of IIT Kanpur. Organised in association with ISB Hyderabad.</h5>
+                    <br><br><br><br><br><br><br><br><br><br><br><br><br>
```
Comment (kumardeepakr3) [inline index.html:136]: This doesn't work when the browser height is less. Check this.

**Review by kumardeepakr3 — CHANGES_REQUESTED** [review]
Comments on PR

File: index.html (hunk 1 above)
Comment (adityaagr99) [reply index.html:136]: okk I will do somthing else then

**Comment by kumardeepakr3** [comment]
Changes done in #4

**Pull request closed without merging**